olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/7.12.1/eclipse-che-preview-openshift.v7.12.1.clusterserviceversion.yaml

```diff
@@ -27,7 +27,7 @@ metadata:
                 "chePostgresDb": ""
              },
              "auth": {
-                "openShiftoAuth": true,
+                "openShiftoAuth": false,
```

Comment:
Why are you changing this in the OLM packages ? Usually we should never change anything in those folders without creating a new nightly version first.

Reply:
@davidfestal Is just for test purpose. I'm trying to install in Centos CI and run test updates. When I'll finish this task obviously I'll not change this files.